src/markdown-pages/docs/ecosystem/deploy-to-maven.md

```diff
@@ -0,0 +1,120 @@
+---
+path: "/docs/ecosystem/deploy-to-maven"
+title: "Deploying a test to a Maven repository"
+---
+
+To run a test in automation, the test engines need access to the test material. This is achieved by offering the test material as Maven artifacts, stored in a Maven repository. If you are using the Docker Operator to bring up an ecosystem, a Nexus repository is also instantiated, which can be used to store the artifacts. However, you can use any Maven repositories to which you have access.
+
+###  Define to Maven where the artifacts are to be deployed.
+This example uses a local Nexus repository that can be brought up locally on a workstation.
+Complete the following steps to define where you want to deploy the artifacts:
+1. Create or edit the ```settings.xml``` that is used by Maven. The default path of ```settings.xml``` is ```~/.m2```.
+    1. Update the details of the Nexus server for _id_, _username_, and _password_.
```

Comment:
"Refer to the maven documentation for details of how to encrypt this password if required"